docs/development.md

```diff
@@ -344,3 +344,145 @@ respectively.
 ```bash
 xtask --logs run-cargo-fuzz --crate-name=loader --target-name=wasm_invoke -- -max_total_time=20
 ```
+
+## Retrieving a previously-built binary and its provenance
+
+Every time a pull-request is merged to `main` the
+`oak_functions_freestanding_bin` binary is built and uploaded to `ent`.
+Similarly, a
+[signed SLSA3+ provenance is generated](https://github.com/slsa-framework/slsa-github-generator/blob/04f1fe0c7b7902c9a95f4c7eef2dc04cf0f8e6a7/internal/builders/generic/README.md)
```

Comment:
super nit: I think "is generated" should be outside of the link.

Reply:
Thanks. Fixed.